contracts/registry/list.sol

```diff
@@ -26,7 +26,11 @@ contract DSMath {
 contract Variables is DSMath {
 
     // InstaIndex Address.
-    address public constant instaIndex = 0x0000000000000000000000000000000000000000;
+    address public immutable instaIndex;
```

Comment:
Why not just keep this constant? Because of hardhat?

Reply:
Because of the hardhat deployment script. It would deploy all the contracts.